R/add.R

```diff
@@ -398,7 +398,7 @@ addMetaFeatures <- function(study, metaFeatures, reset = FALSE) {
 #'   will be assumed to be \code{"singleFeature"} and \code{"singleTest"}.
 #'   Optionally, if the plotting function requires external packages, these can
 #'   be defined in the element \code{packages}. To share plots across multiple
-#'   models, use the modelID "default".
+#'   models, use the modelID "default". This is modified to accept plotly plots.
```

Comment:
Please rephrase this to provide guidance to the user. For example, it could start something like, "To add a plotting function that returns an interactive plotly plot, set the \code{plotType} to include..."